Dockerfile

```diff
@@ -1,20 +1,197 @@
-FROM debian:buster
+FROM statelesstestregistry.azurecr.io/stateless/base:9
 
+# Install necessary Debian packages.
```

Comment:
Can I ask you to update the README.md file with any steps necessary to build trex or its container for node100?

Reply:
Or at least remove from README.md anything not currently relevant. I can help if you're unsure.